pkgs/development/tools/analysis/valgrind/default.nix

```diff
@@ -19,6 +19,10 @@ stdenv.mkDerivation rec {
       url = "https://bugsfiles.kde.org/attachment.cgi?id=143535";
       sha256 = "036zyk30rixjvpylw3c7n171n4gpn6zcp7h6ya2dz4h5r478l9i6";
     })
+    (fetchpatch {
```

Comment:
Might be good to have a comment with a link to the upstream bug report, so it's easy to see how progress upstream is going?